eZ/Bundle/EzPublishCoreBundle/Resources/config/papi.yml

```diff
@@ -98,3 +98,10 @@ services:
             - '@ezpublish.spi.search'
         tags:
             - { name: kernel.event_subscriber }
+
+    ezplatform.api_event_listner.user.delete_user_signal:
```

Comment:
maybe `ezplatform.api.event_listner.user.delete_user_signal` or because this goes to `7.5` even `ezpublish.api.event_listner.delete_user_signal` ?

Reply:
`papi.yml` is not a good choice here, Slot is not a part of PAPI. Slot placement would suggest placing service definition in `eZ/Publish/Core/settings/repository`, but this is meant for Repository layers, so it might be confusing.

Let's create `eZ/Publish/Core/settings/slots.yml` instead and remember to include it both for integration tests (SetupFactory class) and Bundle.

Reply:
What do you say to `slot.yml`? In the ` eZ/Publish/Core/settings/` there is no naming convention. There are files in both plural and singular form. But singular seems to me to be the more correct form.

Reply:
There's more in plural form than singular, but if you prefer singular there, no problem :)